docs/content/rules/sort-switch-case.mdx

```diff
@@ -27,6 +27,10 @@ This practice contributes to a more readable and maintainable codebase, allowing
 
 By integrating this rule into your ESLint configuration, you can focus on the functionality of your code, confident that your switch statements are consistently structured and easy to manage.
 
+<Important>
+It is highly recommended to disable the [`no-useless-return`](https://eslint.org/docs/latest/rules/no-useless-return) rule to avoid conflicts (see [this issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues/348)).
```

Comment:
Thank you. I need time to reproduce locally the problem with `no-useless-return` and `perfectionist/sort-switch-case`.

Reply:
@azat-io No worries, here is a simple configuration if that can save you some time (legacy config with `"eslint": "^8.57.0"`):

.eslintrc.json
```json
{
  "extends": [
    "eslint:all",
    "plugin:@typescript-eslint/strict",
    "plugin:perfectionist/recommended-alphabetical-legacy"
  ]
}
```

Code
```ts
function funct(x) {
    switch(x) {
        case "b":
            break;
        case "a":
            return;
    }
}
```

Reply:
That's an interesting observation. Thank you, the problem is indeed a curious one.

However, it seems to me that this case is very rare, and the `no-useless-return' rule is quite popular. I use it myself in my work. Also this rule is included in popular ESLint configs:

- `eslint-config-standard`
- `eslint-config-airbnb`
- `eslint-config-eslint`
- etc.

Maybe we should remove this warning? Because the case is really rare.

Reply:
@azat-io Good news, I have run additional tests on this version (next) and can't reproduce it, it seems that the previous PR fixed it. I reverted the documentation commit.

Reply:
Alright, I understand why: the issue actually comes from the fact that

```ts
function funct(x) {
    switch(x) {
        case "b":
            let a;
            break;
        case "a":
    }
}
```

will get sorted to

```ts
function funct(x) {
    switch(x) {
        case "b":
            let a;
            break;
        case "a":
    }
}
```

`no-useless-return` doesn't actually affect us! This PR fixes it, I'll add a test.